image/templates/helm/stackrox-secured-cluster/templates/_init.tpl.htpl

```diff
@@ -268,7 +268,22 @@
 {{/*
    Local Scanner v4 Indexer setup.
   */}}
+
 {{ $scannerV4Cfg := $._rox.scannerV4 }}
+
+{{/* In order to establish a smooth and automated activation of Scanner v4 in situations where Scanner v2 is
```

Comment:
super nit: You can remove `In order`

Reply:
I will try to remember this. Dunno why it is so pervasive in my english vocabulary.